dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -759,5 +759,70 @@ public void VerifyNavigationViewItemInFooterDoesNotCrash()
                 Verify.IsTrue(true);
             });
         }
+
+        [TestMethod]
+        public void VerifyExpandCollapseChevronVisibility()
+        {
+            NavigationView navView = null;
+            NavigationViewItem parentItem = null;
+            ObservableCollection<string> children = null;
+
+            RunOnUIThread.Execute(() =>
+            {
+                navView = new NavigationView();
+                Content = navView;
+
+                children = new ObservableCollection<string>();
+                parentItem = new NavigationViewItem() { Content = "ParentItem", MenuItemsSource = children };
+
+                navView.MenuItems.Add(parentItem);
+
+                navView.Width = 1008; // forces the control into Expanded mode so that the menu renders
+                Content.UpdateLayout();
+
+                UIElement chevronUIElement = (UIElement)FindVisualChildByName(parentItem, "ExpandCollapseChevron");
```

Comment:
We can use `VisualTreeUtils.FindVisualChildByName()` here instead of having to provide an own implementation.

Reply:
I also removed the FindVisualChildByName method.